applications/nrf_desktop/README.rst

```diff
@@ -810,6 +810,38 @@ The nRF Desktop application is built the same way to any other |NCS| application
 .. note::
     Information about the known issues in nRF Desktop can be found in |NCS|'s :ref:`release_notes` and on the :ref:`known_issues` page.
 
+Adding nRF21540EK shield support
+================================
+
+The nRF21540EK contains RF front-end module (FEM) for 2.4 GHz range extension.
+Any application can be built with the shield support that was already implemented
+just passing the SHIELD parameter to the built command.
+
+The build command for nRF21540EK would look differently for single and multicore configuration.
+
+Single core application
+-----------------------
+
+In single core application the single SHIELD parameter needs to be passed to the build command.
+For example for nRF52840DK paired with nRF21540EK it will be:
+
+    .. code-block:: console
+
+       west build -b nrf52840dk_nrf52840 -- -DSHIELD=nrf21540_ek
+
+Multicore application
+---------------------
+
+In multiple core application, where bluetooth runs on other than application core both cores needs to know about the FEM shield implemented.
```

Comment:
Bluetooth Controlller ? The host still runs on the app core

Reply:
This comment is obsolete as whole sentence was removed. Now we are rely on the linked documentation.